### PR TITLE
[FIX] Add intermediate step for Sequence

### PR DESCRIPTION
**Pull request by c-py**
This adds back the intermediate step for sequence.

This is required so that a pop up has a better chance of showing up in chrome.

### CODE REVIEW

File: src/lib/blockchain/wallet.ts [hunk 1]
```diff
@@ -211,7 +211,7 @@ export class Wallet {
           symbol: "MATIC",
           decimals: 18,
         },
-        rpcUrls: ["https://matic-mumbai.chainstacklabs.com"],
+        rpcUrls: ["https://rpc.ankr.com/polygon_mumbai"],
```
Comment (c-py) [inline src/lib/blockchain/wallet.ts:214]: Chainstacklabs seems to be down at the moment

Comment (adamhannigan) [inline src/lib/blockchain/wallet.ts:214]: It would be nice if we had 3-4 fallback URLs to cycle through ;)

**Review by adamhannigan — APPROVED** [review]
LGTM ✅

File: src/lib/blockchain/wallet.ts (hunk 1 above)
Comment (c-py) [reply src/lib/blockchain/wallet.ts:214]: This is a good idea 👍

**Pull request merged**